File: app/src/main/java/com/filipkin/ftahelper/ui/monitor/MonitorViewModel.java
package com.filipkin.ftahelper.ui.monitor;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class MonitorViewModel extends ViewModel {
    private MutableLiveData<FieldState> field;

    public MutableLiveData<FieldState> getField() {
        if (field == null) {
            field = new MutableLiveData<>();
        }

        return field;
    }
}

File: app/src/main/java/com/filipkin/ftahelper/ui/monitor/MonitorViewModel.kt
package com.filipkin.ftahelper.ui.monitor

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class MonitorViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is dashboard Fragment"
    }
    val text: LiveData<String> = _text
}